framework/Optimizers/SPSA.py

```diff
@@ -422,33 +463,6 @@ def clearCurrentOptimizationEffort(self,traj):
     except KeyError:
       pass
 
-  #def estimateStochasticity(self,gradient,perturbedPoints,centralPoint,centralResponseIndex):
-  #  """
-  #    TODO current unused, so commented, but will be used in future patch.
```

Comment:
I looked through it and it needs to be reworked sufficiently that it would be better started from scratch.  In the event we can't do better (extremely unlikely), this will still be available in the history.